selenium/pom.xml

```diff
@@ -1,215 +1,224 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <artifactId>selenium</artifactId>
-  <version>4.7.0</version>
-  <packaging>jar</packaging>
-  <parent>
-    <groupId>com.deque.html.axe-core</groupId>
-    <artifactId>virtual-superpackage</artifactId>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>selenium</artifactId>
     <version>4.7.0</version>
```

Comment:
Should this be 4.8.0?

Reply:
https://github.com/dequelabs/axe-core-maven-html/pull/398